GPU/TPCFastTransformation/Spline2D.cxx

```diff
@@ -163,7 +250,7 @@ int Spline2D::test(const bool draw, const bool drawDataPoints)
   const int nAxiliaryPoints = 1;
   int uMax = nKnots * 3;
 
```

Comment:
should not the `TMath::Pi() / (nKnots * 3)` be precalculated as a constexpr?

Reply:
ok, I fixed it. It is just a test method, though. Optimizing it doesn't worth the effort.

Reply:
ok, did not know it is a test, sorry for silly suggestion